lib/builder-webpack5/src/index.ts

```diff
@@ -138,7 +138,11 @@ export const build: WebpackBuilder['build'] = async ({ options, startTime }) =>
           errors.forEach((e) => logger.error(e.message));
           warnings.forEach((e) => logger.error(e.message));
 
-          compiler.close(() => fail(stats));
+          compiler.close(() =>
+            options.debugWebpack
+              ? fail(stats)
+              : fail(new Error('=> Webpack failed, learn more with --debugWebpack'))
```

Comment:
```suggestion
              : fail(new Error('=> Webpack failed, learn more with --debug-webpack'))
```

Reply:
You would think I would get a 2 line change right. Good catch - done.